lib/archethic_web/api/jsonrpc/methods/estimate_transaction_fee.ex

```diff
@@ -19,8 +19,8 @@ defmodule ArchethicWeb.API.JsonRPC.Method.EstimateTransactionFee do
   """
   @spec validate_params(param :: map()) ::
           {:ok, params :: Transaction.t()} | {:error, reasons :: list()}
-  def validate_params(params) do
-    case TransactionPayload.changeset(params) do
+  def validate_params(%{"transaction" => transaction_params}) do
+    case TransactionPayload.changeset(transaction_params) do
```

Comment:
It could be nice to return an error if `transaction_params` is not a map.
This should be done in `TransactionPayload` I think